src/App.js

```diff
@@ -1,25 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
+import './index.css';
 
 function App() {
```

Comment:
- [ ] The App.js component should be class-based. I kindly invite you to work on it. 🐸

Reply:
hello Victor I don't know what is the problem with the linters but they work fine on my PC I did a commit now can you tell me if this fixes  the linter problem